src/components/post-details-page/PostDetailsPage.js

```diff
@@ -1,34 +1,52 @@
 import React, { Component } from 'react';
+import {accessToken} from "../../constants";
+import {PostCard} from "../post-card/PostCard";
 
 class PostDetailsPage extends Component {
 
   constructor(props) {
     super(props);
 
     this.state = {
-      post: null
+      post: null,
       // todo 3: добавить isLoading флажок - индикатор загрузки
+      isLoading: false
     };
   }
 
   componentDidMount() {
     // todo 3: вызвать загрузку инфрмации про пост loadPost
+    this. loadPost();
   }
 
   loadPost = async () => {
     // todo 3:
     //  достать id поста из props посредством пропсов, которые дает нам роутер
+    const {match: {params: {Id}}} =  this.props;
     //  проверьте лежит ли ваш accessToken в constants/index.js
     //  прежде чем отправить запрос - включите в true флажок загрузки в стейте isLoading
+    await this.setState({
+      isLoading: true
+    });
     //  выполните fetch запрос за поcтом на `https://gorest.co.in/public-api/posts/${id}?access-token=${accessToken}`
     //  похожий запрос выполнялся в компоненте PostCard в функции loadComments
     //  результат выполнения запроса нужно положить в стейт в post
+    await fetch(`https://gorest.co.in/public-api/posts/${Id}?access-token=${accessToken}`)
+        .then(response => {if (response.ok){
+          return response.json()
+        }})
+        .then(json => {if (typeof json.result === 'object') this.setState({
+          post: json.result
```

Comment:
Here you also can change loading together with post
````
this.setState({
          post: json.result,
isLoading: false,
})
````

And you will not need code on lines 42-44

Reply:
A state is an object, you can put such many properties as you need in 1 setState